dashboard/src/containers/helpers/index.ts

```diff
@@ -7,7 +7,5 @@ import { IKubeState } from "shared/types";
 // found are filtered out and the response will only contain resources that are
 // available in the state.
 export function filterByResourceRefs(refs: ResourceRef[], resources: IKubeState["items"]) {
-  return refs
-    .map(r => resources[keyForResourceRef(r.apiVersion, r.kind, r.namespace, r.name)])
-    .filter(r => r !== undefined);
+  return refs.map(r => resources[keyForResourceRef(r)]).filter(r => r !== undefined);
```

Comment:
I don't know about our preference of using `lodash` vs plain js, but I guess you can also do sth like `_.omitBy`. Anyway, it is clear enough and easy to understand, so that's totally fine with me.

Reply:
Yeah, this was the existing code that needed to handle undefined resources in the state. I'm not sure how the state could previously get into this situation, but I'm pretty confident that it cannot do so now anyway (as we only add resources to the state in the first place if they are sent by the resources plugin), but lots of test data currently depends on it. I may remove it in another PR and update the tests.